OWNERS

```diff
@@ -1,24 +1,15 @@
 approvers:
-  - fntlnz
-  - leodido
-  - kris-nova
-  - mfdii
-  - lucperkins
-  - radhikapc
   - leogr
-  - Rajakavitha1
   - jasondellaluce
   - Issif
 reviewers:
+  - danpopsd
```

Comment:
> Hey Jason, `danpopsd` is now `danpopnyc`, could you fix it?
> 
> Also, this PR is blocked by because of reviewers' "change request"s. I'd suggest opening a new one.

Closing this in favor of #655.

/close